evalai/utils/teams.py

```diff
@@ -0,0 +1,136 @@
+import json
+import os
+import requests
+import sys
+
+from click import echo, style
+
+from evalai.utils.auth import get_request_header
+from evalai.utils.common import validate_token
+from evalai.utils.urls import URLS
+
+
+EVALAI_ERROR_CODES = [400, 401, 406]
+
+API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')
+
+
+def pretty_print_teams(teams):
+    """
+    Pretty prints the teams
+    """
+    for team in teams:
+        br = style("----------------------------------------"
+                   "--------------------------", bold=True)
+
+        team_name = "\n{}".format(style(team["team_name"],
+                                        bold=True, fg="green"))
+        team_id = "ID: {}\n\n".format(style(str(team["id"]),
+                                            bold=True, fg="blue"))
+        title = "{} {}".format(team_name, team_id)
+        created_by = "Created by : {}\n\n".format(style(team["created_by"], fg="blue"))
```

Comment:
Can we please change the color from blue. It doesn't appears to be very clear and good.